### PR TITLE
Feat/navigation

### PR DESCRIPTION
**Pull request by Jeysieeeee**

### CODE REVIEW

File: Style.tsx [hunk 1]
```diff
@@ -63,7 +62,7 @@ export const styles = StyleSheet.create({
     marginLeft: 0,
   },
   loginTextContainer: {
-    marginTop: 130,
+    // marginTop: 130,
```
Comment (edwinpergis) [inline Style.tsx]: remove unused code

File: app/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { StyleSheet, Text, View } from "react-native";
+import OnBoarding from "../screens/OnBoarding";
+
+export default function Page() {
```
Comment (edwinpergis) [inline app/index.tsx]: use arrow function

File: app/tabs/message.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Text } from "react-native";
+
+export const message = () =>
```
Comment (edwinpergis) [inline app/tabs/message.tsx]: function and component name should start big letter

File: app/tabs/notification.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { Text } from "react-native";
+
+export const notification = () =>
```
Comment (edwinpergis) [inline app/tabs/notification.tsx]: function and component name should start big letter

File: app/tabs/record.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { Text } from "react-native";
+
+export const record = () =>
```
Comment (edwinpergis) [inline app/tabs/record.tsx]: function and component name should start big letter

**Review by edwinpergis — CHANGES_REQUESTED** [review]

**Review by edwinpergis — APPROVED** [review]

**Review by jarapamikaella — APPROVED** [review]

**Pull request merged**